constraint/pkg/client/rego_helpers.go

```diff
@@ -143,10 +149,16 @@ type ruleArities map[string]int
 
 // requireRules makes sure the listed rules are specified with the required arity
 func requireRules(name, rego string, reqs ruleArities) error {
+	if rego == "" {
+		return errors.New("Rego source code is empty")
+	}
 	module, err := ast.ParseModule(name, rego)
 	if err != nil {
 		return err
 	}
+	if module == nil {
+		return "", errors.New("Module could not be parsed")
```

Comment:
oh this is giving us an error. should just return error, no string

Reply:
oops, that's what I get for not using an IDE...

fixed, will push if tests pass.